<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenFlowFilesBar</class>
 <widget class="QWidget" name="OpenFlowFilesBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>609</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vl_mainlayout">
    <item>
     <layout class="QGridLayout" name="gl_inputfiles">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lb_inputmesh">
          <property name="text">
           <string>Mesh File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_meshfile">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lb_tolerance">
        <property name="text">
         <string>Tolerance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="edt_inputparameters"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btn_inputmesh">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="Line" name="ln_inputfiles_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lb_inputparameters">
        <property name="text">
         <string>Parameters File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="edt_inputmesh"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_inputparameters">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="Line" name="ln_inputfiles">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="edt_tolerance1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="edt_tolerance2"/>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_defaulttolerances">
          <property name="text">
           <string>Default</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="rd_cornerpoint">
            <property name="text">
             <string>Corner Point </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rd_unstructuredmesh">
            <property name="text">
             <string>Unstructured Mesh</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="btb_inputfiles">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
